src/classes/Metrics.js

```diff
@@ -270,8 +270,6 @@ class Metrics {
 			buildQueryPair('at', conf.enable_anti_tracking ? '1' : '0') +
 			// The deepest setup page reached by user during setup
 			buildQueryPair('ss', Metrics._getSetupStep(type).toString()) +
-			// The number of times the user has gone through setup in the regular hub, or the answer selection on each page of the dawn-hub
-			buildQueryPair('sl', Metrics._getSetupNumber(type).toString()) +
```

Comment:
I think we should leave this in but change back to 
`buildQueryPair('sl', conf.setup_number.toString()) +`

Reply:
I wasn't sure about that one, was going to check AWS Glue if I would not break the schema. Thanks for help Chris!